glacierscript.py

```diff
@@ -341,6 +342,18 @@ def get_utxos(tx, address):
 
     return utxos
 
+def verbose(content):
+    # if verbose mode enabled, print content
+    if VERBOSE_MODE:
+        print content
+
+def bitcoin_cli_call(cmd,args):
+    full_cmd = "{0}{1} {2}".format(bitcoin_cli,cmd,args)
+    # note glacier has a space after bitcoind call in "bitcoin_cli" variable
```

Comment:
Why not remove that space now, and move it to full_cmd? It was only there to make the usage of it a little cleaner, but that's gone now.

Reply:
Will do...I'll incorporate into my base-upgrades branch. Thank you!

Reply:
In my development branch I've taken out the space in the bitcoin cli & related arg variables and moved the space into this bitcoin_cli_call function. To not interfere with other bitcoin-cli calls (these are handled differently re e.g. shell options), I've added the space to these other calls. Thanks!